aten/src/ATen/core/op_registration/op_registration.h

```diff
@@ -398,9 +398,23 @@ class CAFFE2_API RegisterOperators final {
       return std::move(*this).op(schemaOrName, std::move(options).kernelFunctor<detail::WrapRuntimeKernelFunctor<guts::decay_t<FuncType>>, AllowLegacyTypes>(c10::nullopt, std::forward<FuncType>(func)));
     }
 
+    /**
+     * This API registers an operator based on a kernel lambda.
+     *
+     * This API looks like:
+     *
+     * > static auto registry = c10::RegisterOperators()
+     * >     .wrapper("{schema}", &my_variable_wrapper);
+     */
+    RegisterOperators wrapper(const std::string& schemaOrName, KernelFunctionWrapper* wrapper) {
```

Comment:
let's (a) say what kind of wrapper and (b) make it more scary so external people don't use it yet. Something like `_internalAutogradWrapper`?

Reply:
Also, let's make this a method on `OperatorOptions` as opposed to on `RegisterOperators`. That avoids having to specify the schema twice.

Instead of:

    auto registrar = c10::RegisterOperators()
        .op("_test::dummy(Tensor dummy) -> ()",
            c10::RegisterOperators::options()
                .kernel<DummyKernel>(TensorType1()))
        .wrapper("_test::dummy(Tensor dummy) -> ()", &dummy_autograd);

that would become:

    auto registrar = c10::RegisterOperators()
        .op("_test::dummy(Tensor dummy) -> ()",
            c10::RegisterOperators::options()
                .kernel<DummyKernel>(TensorType1())
                .wrapper(&dummy_autograd));